app/assets/stylesheets/themes/default/pages/_matches.scss

```diff
@@ -193,16 +181,9 @@ div#match {
         margin: 0 0 0 10px;
       }
 
-      .controls {
-        left: 0;
-      }
-
-      form {
-
-        label.checkbox {
-          float: right;
-          margin-left: 1em;
-        }
+      label.checkbox {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4